src/api/app/models/package.rb

```diff
@@ -120,6 +116,23 @@ def self.check_access?(dbpkg = self)
     return Project.check_access?(dbpkg.project)
   end
 
+  def self.dirty_backend_packages
+    # avoid scope here, it will turn objects readonly
```

Comment:
You're doing this to avoid the default scope? Please use [unscoped](http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Default/ClassMethods.html#method-i-unscoped) for this...